client/client.go

```diff
@@ -8,9 +8,9 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/client-go/rest"
 	clusterv1client "open-cluster-management.io/api/client/cluster/clientset/versioned"
-	clusterv1beta2 "open-cluster-management.io/api/cluster/v1beta2"
 	"open-cluster-management.io/cluster-proxy/pkg/generated/clientset/versioned"
 	"open-cluster-management.io/cluster-proxy/pkg/util"
+	clustersdk "open-cluster-management.io/sdk-go/pkg/apis/cluster/v1beta2"
```

Comment:
@qiujian16 
Looking for advise for this one, we seems having the 'clusterv1beta2' name style for importing a pkg for a long time and across repos.  When using a api lib, do we need to specify the version?

Reply:
yes, I think so

Reply:
Updated